multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -187,129 +207,136 @@ test.serial('lp deposits', async t => {
   );
 });
 
-test.serial('advance and settlement', async t => {
-  const {
-    nobleTools,
-    nobleAgoricChannelId,
-    oracleWds,
-    retryUntilCondition,
-    useChain,
-    usdcOnOsmosis,
-    vstorageClient,
-  } = t.context;
-
-  // EUD wallet on osmosis
-  const eudWallet = await createWallet(useChain('osmosis').chain.bech32_prefix);
-  const EUD = (await eudWallet.getAccounts())[0].address;
-
-  // parameterize agoric address
-  const { settlementAccount } = await vstorageClient.queryData(
-    `published.${contractName}`,
-  );
-  t.log('settlementAccount address', settlementAccount);
+const advanceAndSettleScenario = test.macro({
+  title: (_, mintAmt: bigint, eudChain: string) =>
```

Comment:
`eudChain` is kinda... tortured

```suggestion
  title: (_, mintAmt: bigint, destChain: string) =>
```